src/style/font/font_desc.rs

```diff
@@ -255,7 +256,7 @@ impl<'a> FontDesc<'a> {
         TextStyle {
             font: self.clone(),
             color: color.to_rgba(),
-            alignment: TextAlignment::Left,
+            pos: Pos::default(),
```

Comment:
Not about this one, but I believe after the change, we can get rid of `FontTransform::offset` right? Because we have better model that rotating the text based on the anchor point as the center.

Reply:
Yes, I'll remove it.